package.json

```diff
@@ -137,5 +134,12 @@
     "sinon": "7.2.3",
     "validate-commit-msg": "2.14.0",
     "webpack": "4.29.3"
+  },
+  "husky": {
+    "hooks": {
+      "commit-msg": "validate-commit-msg",
+      "pre-commit": "lint-staged",
+      "pre-push": "run-p lint test"
```

Comment:
no pre-publish ?

Reply:
Nice detection! I don't think it's a git hook https://git-scm.com/docs/githooks however. @iamstarkov maybe you know more.

Reply:
`prepublish` as in `{pre,post}anyNpmScriptName` is a script to be invoked implicitly by npm right before it will invoke publish

Reply:
it is not a git hook